src/reactors.js

```diff
@@ -33,7 +33,6 @@ let reactors = {
 
 export let StandardReactors = {
   refresh: reactors.all,
-  data: reactors.others,
```

Comment:
If we're doing this, we should remove config as well, but it's probably safe to leave